### PR TITLE
update a blog post and FQA for HTTPS

### PR DESCRIPTION
**Pull request by haoliangyu**

### CODE REVIEW

File: _posts/2021-10-19-koop-cli-v1-1-0-release.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Koop CLI v1.1 release: add HTTPS support to the dev server"
+date:   2021-01-19
+author: Haoliang Yu
+---
+
+HTTPS support is one of the mostly asked features for Koop. In the recent release of Koop CLI v1.1, we have added the support of HTTPS to the dev server, which allows to create a local HTTPS server with simple command options. In this blog, we will discuss this new feature.
```
Comment (rgwozdz) [inline _posts/2021-10-19-koop-cli-v1-1-0-release.md:8]: `mostly asked` -> `most requested`

File: _posts/2021-10-19-koop-cli-v1-1-0-release.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Note that the URL protocol now is `https`, instead of `http`. Such URL is ready to be used in the client that requires HTTPS links.
+
+## Under the hood
+
+The `server` command uses the native node moudle [https](https://nodejs.org/api/https.html) to create the HTTPS server. You can see the exact implementation in the office guideline [How to create an https server?](https://nodejs.org/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/).
+
+## What about production?
+
+As the whole blog post is talking about the dev server, you may ask what about the production. Since Koop is a dedicated ET(L) server, it does not incldue the HTTPS feature. The implementation for the HTTPS dev server is not scalable and is not a good way for production use.
```
Comment (rgwozdz) [inline _posts/2021-10-19-koop-cli-v1-1-0-release.md:54]: change to `you may ask "What about the production?"`

**Review by rgwozdz — APPROVED** [review]
a few suggestions

**Comment by haoliangyu** [comment]
Thank you for the suggestion!

**Pull request merged**